components/searchable_channel_list.jsx

```diff
@@ -69,7 +68,7 @@ export default class SearchableChannelList extends React.PureComponent {
         if (shouldShowArchivedChannels) {
             archiveIcon = (
                 <div className='more-modal__icon-container'>
-                    <ArchiveIcon className='icon icon__archive'/>
+                    <i className='icon icon-archive-outline'/>
```

Comment:
@zefhemel @hmhealey We are using a compass component at some places and an `i` tag at some. Not sure if we would like to recommend people to use compass components, but This seems a good opportunity to discuss that at least for icons
cc// @Willyfrog @jwilander
https://github.com/mattermost/mattermost-webapp/blob/b5e57eb863b71a41f0b9b2ba0b8e517b27486dc2/components/file_upload/file_upload.jsx#L8
 https://github.com/mattermost/mattermost-webapp/blob/b5e57eb863b71a41f0b9b2ba0b8e517b27486dc2/components/file_upload/file_upload.jsx#L612

Reply:
That's a good question. I think they're in a spot where we can recommend using them, but I'll check with @michelengelen just in case.

Reply:
Yeah, we should be recommending the compass-icons package going forward

Reply:
@thinkGeist Icon list https://mattermost.github.io/compass-icons/

Reply:
Cool, thanks for the feedback. I've adjusted the change to leverage the icon from from the compass-icons package instead, with an updated color to match the `--center-channel-color-rgb` in both instances for this issue (though there is still open discussion about which variable to use)

Thanks again for the review!

Reply:
Just for clarification: the `@mattermost/compass-icons` package is fully supported and the `@mattermost/compass-components` package is not (at the moment), so using icon components from `@mattermost/compass-icons/components` is 100% viable and should be pushed forward.

@thinkGeist most of the time you can just use `currentColor` since the correct color should be applied to the container anyways (it probably is `rgba(var(--center-channel-color-rgb, 1)` in most cases). Thats the easiest approach. You should check if the correct color is applied though for cases where the icon lives outside of a container with a defined font-color.